Godeps/Godeps.json

```diff
@@ -2345,8 +2345,8 @@
 		},
 		{
 			"ImportPath": "github.com/json-iterator/go",
-			"Comment": "1.1.3-22-gf2b4162afba355",
-			"Rev": "f2b4162afba35581b6d4a50d3b8f34e33c144682"
+			"Comment": "1.1.4",
+			"Rev": "ab8a2e0c74be9d3be70b3184d9acc634935ded82"
```

Comment:
I'm sure I'm probably missing something here but doesn't the commit https://github.com/json-iterator/go/commit/ab8a2e0c74be9d3be70b3184d9acc634935ded82 contain code changes?

Reply:
It changes things in the `github.com/json-iterator/go/extra` package, but that's not imported/vendored by the main `github.com/json-iterator/go` package